sssd_test_framework/roles/ipa.py

```diff
@@ -641,6 +641,202 @@ def passkey_remove(self, passkey_mapping: str) -> IPAUser:
         self._exec("remove-passkey", [passkey_mapping])
         return self
 
+    def iduseroverride(self) -> IDUserOverride:
+        """
+        Add override to the IPA user.
+
+        .. code-block:: python
+            :caption: Example usage
+
+            @pytest.mark.topology(KnownTopology.IPA)
+            def test_example(client: Client, ipa: IPA):
+                ipa.idview("newview1").add(description="This is a new view")
+                ipa.idview("newview1").apply(hosts="client.test")
+                ipa.user("user-1").add().iduseroverride().add_override("newview1", uid=1344567)
+                client.sssd.restart()
+                lookup1 = client.tools.id("user-1")
+                assert lookup1.user.id == 1344567
+
+        :return: New IDOverride object.
+        :rtype: IDOverride
+        """
+        return IDUserOverride(self)
+
+
+class IDUserOverride(IPAUser):
```

Comment:
In https://github.com/SSSD/sssd-test-framework/pull/146 you have `class IPAIdView(IPAObject):` but here you have `class IDUserOverride(IPAUser):`

I suggest using the same upper or lower case convention  either `ID`, or `Id`

Reply:
Keeping `ID` as it looks more readable, updating #146 PR.

Reply:
Updated #146 also.